components/org.wso2.is.password.migrate/src/main/java/secondaryUserStore/internal/ISMigrationServiceDataHolder.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package secondaryUserStore.internal;
+
+import org.wso2.carbon.base.api.ServerConfigurationService;
+import org.wso2.carbon.identity.claim.metadata.mgt.ClaimMetadataManagementService;
+import org.wso2.carbon.registry.core.service.RegistryService;
+import org.wso2.carbon.registry.core.service.TenantRegistryLoader;
+import org.wso2.carbon.user.core.service.RealmService;
+
+/**
+ * Migration Service Data Holder
+ */
+public class ISMigrationServiceDataHolder {
+
+    //Registry Service which is used to get registry data.
+    private static RegistryService registryService;
```

Comment:
remove unused references. you should be needing only ServerConfigurationService reference.

Reply:
okay